ng-dev/pr/common/fetch-pull-request.ts

```diff
@@ -81,6 +82,16 @@ export const PR_SCHEMA = {
   reviewRequests: {
     totalCount: graphqlTypes.number,
   },
+  reviews: params(
+    {last: 100, states: 'APPROVED'},
```

Comment:
😄 this is still not ideal, but we are good in practice haha

Reply:
I don't even know what would happen in Github if more than 100 people approve a PR, I feel like it would break anyway.